lib/ansible/module_utils/k8svirt/helper.py

```diff
@@ -379,3 +395,81 @@ def replace(self, body, namespace, name):
         return (self.__client.
                 replace_namespaced_virtual_machine_instance_preset(
                     vmips_body, namespace, name))
+
+
+class PersistentVolumeClaimHelper(object):
+    """ Helper class for PersistentVolumeClaim resources """
+
+    def __init__(self, client):
+        """ Class constructor """
+        self.__client = client
+
+    def create(self, body, namespace, wait=False, timeout=300, interval=3):
+        """ Create PersistentVolumeClaim resource """
+        pvc_body = core_sdk.V1PersistentVolumeClaim().to_dict()
+        pvc_body.update(copy.deepcopy(body))
+        created_pvc = self.__client.create_namespaced_persistent_volume_claim(
+            namespace=namespace, body=pvc_body)
+        if not wait:
+            return created_pvc
+
+        name = pvc_body['metadata']['name']
+        start = time.time()
+        while time.time() < start + timeout:
+            entity = self.__client.read_namespaced_persistent_volume_claim(
+                name, namespace)
+            if entity.status.phase == 'Succeeded':
```

Comment:
@pkliczewski what is the expected status in this context ? According to https://github.com/ansibleplaybookbundle/import-vm-apb/blob/master/roles/import-from-url/tasks/provision.yml#L66 it should be Bound.

Reply:
Based on the logic from `import-vm-apb` we need to wait on status `Bound` and later on import_status to be either 'Succeeded' or 'Failed'. This makes it a bit more complicated because first we wait on creation and later on import. That is why I suggested to have a separate waiting task like in `import-vm-apb`.